tests/test_bazel_integration.py

```diff
@@ -124,7 +124,7 @@ def test_dot_genproj(self):
         command = GenProjCommandLine()
         command.main(args)
 
-        self.verify_genproj(app_path, 342, 1012, 0)
+        self.verify_genproj(app_path, 342, 675, 0)
```

Comment:
It is now expected we have less swift files because now we include imports and function call lines between them.